File: app/src/main/java/in/nisb/nisbapp/AboutActivity.java
package in.nisb.nisbapp;

import android.graphics.Color;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        ExtraFunctions.setSBColor(getWindow(), Color.parseColor("#95a5a6"));

    }
}

File: app/src/main/java/in/nisb/nisbapp/activities/AboutActivity.kt
package `in`.nisb.nisbapp

import android.graphics.Color
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)

        ExtraFunctions.setSBColor(window, Color.parseColor("#95a5a6"))

    }
}
